.github/workflows/release.yml

```diff
@@ -39,3 +39,17 @@ jobs:
           cd src/github.com/vmware-tanzu/velero-plugin-for-vsphere
           echo ${{ secrets.DOCKERHUB_PASSWORD }} | docker login -u ${{ secrets.DOCKERHUB_USERNAME }} --password-stdin
           make release QUALIFIED_TAG=${{ github.event.release.tag_name }} RELEASE_TAG=${{ github.event.release.tag_name }}
+      # Use the JSON key in secret to login gcr.io
+      - uses: 'docker/login-action@v1'
+        with:
+          registry: 'gcr.io' # or REGION.docker.pkg.dev
+          username: '_json_key'
+          password: '${{ secrets.GCR_SA_KEY }}'
+      # Push image to GCR to facilitate some environments that have rate limitation to docker hub, e.g. vSphere.
```

Comment:
Can you change the comments to the following:
`Push image to GCR to avoid docker hub rate limitation issue.`

Reply:
Sure.